.github/workflows/require-labels.yml

```diff
@@ -0,0 +1,25 @@
+name: 🚨 Check PR Labels
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      pull-requests: read
+      statuses: write
+    outputs:
+      status: ${{ steps.check-labels.outputs.status }}
+    steps:
+      - id: check-labels
+        uses: mheap/github-action-required-labels@v2
+        with:
+          mode: exactly
+          count: 1
+          labels: "Added, Changed, Fixed"
+          exit_type: success
```

Comment:
We'd need to allow the other non-changelog labels too right? (CI, etc)

Reply:
My thinking was that even release-note-excluded changes can fit into one of these 3 categories.
My 🧠
- Maintain baseline requirements: always
- Add exceptions as an additional layer: can be tweaked later on without disrupting our baseline truth

Reply:
Que 👍 or 👎

Reply:
Yeah I guess that makes sense. We can always change it later if we need.